website/docs/r/event_orchestration_unrouted.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_unrouted"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-unrouted"
+description: |-
+  Creates and manages an Unrouted Orchestration for a Global Event Orchestration in PagerDuty.
+---
+
+# pagerduty_event_orchestration_unrouted
+
+An Unrouted Orchestration allows users to create a set of Event Rules that will be evaluated against all events that don't match any rules in the Orchestration's Router.
+
+The Unrouted Orchestration evaluates events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Unrouted Orchestration for further processing.
+
+## Example of configuring a Unrouted Rules for an Orchestration
+
+  In this example the user has defined the unrouted orchestration with two rules, each routing to a different service.This example assumes services used in the route_to configuration already exists. So it does not show creation of service resource.As there is a single set, route_to is not defined for the rules.
+
+  If there are more than one set, the rules in start set must define route_to with id of the next set
+  The first rule matches only if the condition is matched. The second rule matches always as there are no conditions.
+
+  `catch_all` with empty `actions` block suppresses the alerts that do not match any rules. Using the `catch_all` to set severity, event_action, variables and extractions is also allowed.
```

Comment:
**Blocker:** Again, I don't think suppress-by-default is correct (see my comment above).

Also, looking over this example (which is the same as [the API Docs example](https://developer.pagerduty.com/api-reference/953ddc1761403-update-the-unrouted-orchestration-for-a-global-event-orchestration)) makes me realize we have a bug! Seems like we should just use the `catch_all` block _instead_ of creating a rule with no conditions! In fact, that 2nd condition-less rule means the catch_all will **never** be used!

Might be worth changing this example to actually use the `catch_all` instead of 2 rules just to show how it can be used?

Reply:
Good call out. It is true that for service path, the catch all behaviour leaves event as-is. But for `unrouted` the `catch all `always suppresses alert.
<img width="1422" alt="image" src="https://user-images.githubusercontent.com/3372372/169693979-71314b28-74ef-47d7-8572-0c5b69889416.png">
But I do agree that the example is confusing in the sense that 2nd rule will always match so catch_all will never be hit. I have removed the 2nd rule and moved it's alert action to the `catch_all`